fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -438,22 +438,27 @@ def self.run_command(command, timeout: 0, retries: 0, print: true)
 
       total_tries = retries + 1
       try = 1
+      try_timeout = timeout
```

Comment:
maybe document the method with the fact that the timeout will be doubled at each retry?